aa_stripe/settings.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+"""
+Settings for aa-stripe are all namespaced in the STRIPE_SETTINGS setting. For example your project's "settings.py" file
+might look like this:
+
+STRIPE_SETTINGS = {
+    "PENDING_EVENTS_THRESHOLD": 20
+}
+
+To simplify overriding those settings they have a flat structure.
```

Comment:
Flat structure? please explain.

Reply:
Well, this is based on settings from chat_api (which was based on API settings from Rest Framework, right @pkrzyzaniak?). I will change it to have flat settings as you requested in the other PR.